trusted-firmware-m/platform/ext/common/tfm_platform.c

```diff
@@ -13,7 +13,11 @@
 __WEAK enum tfm_hal_status_t tfm_hal_platform_init(void)
 {
     __enable_irq();
+
+    /* Only if UART1 is used by TF-M do we initialize it. */
+#ifdef SECURE_UART1
     stdio_init();
+#endif
```

Comment:
I thought we weren't going to use the __WEAK implementation anymore, so no need to patch it?

Reply:
True, mixed up this file. Now only the removal of the lines in the preload file is there.